content/millennium/r4/individuals/practitioner.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Practitioner | R4 API
+---
+
+# Practitioner
+
+* TOC
+{:toc}
+
+## Overview
+
+The Practitioner Resource provides information about a person formally involved in the care of a patient on behalf of a healthcare facility. Practitioners include but are not limited to physicians, nurses, pharmacists, therapists, technologists, and social workers.
+
+The following fields are returned if valued:
+
+  *	[Practitioner id](https://hl7.org/fhir/r4/resource-definitions.html#Resource.id){:target="_blank"}
```

Comment:
no leading 2-space indent

use 1 space instead of 1 tab between * and words

Reply:
not sure why it rendered in github with a tab when it was already just a space, but removing the indent seemed to fix it: https://github.com/cerner/fhir.cerner.com/pull/252/commits/96a3eb3a205e6dd3f0a70297cd275eb26475b8f1#diff-00a306a2aa1334ff6f799b4200c75030L16